pkg/services/ngalert/notifier/channels/email_test.go

```diff
@@ -53,7 +53,7 @@ func TestEmailNotifier(t *testing.T) {
 			Name:     "ops",
 			Type:     "email",
 			Settings: settingsJSON,
-		}, externalURL)
+		}, externalURL, "")
 
 		require.NoError(t, err)
```

Comment:
I think we could use a holistic assertion here. I'm always weary of tests that don't fail whenever you make a change in the payload - I would have expected this test to fail with the before made change. 

Could we create a test handler in the bus to make sure what we expected has changed?

Reply:
I think this stuff would be a bit easier to test with a subroutine which prepared the payload before dispatching to the bus. Then, we could just test the subroutine and be reasonably confident considering the payload is as expected.

Reply:
The Grafana bus allows us to "inject" mock receivers which help us examine the payload that was sent to the bus - same concept different approach 😄